### ISSUE TITLE
Why after train and the eval results in VOC 2007 just 14(18 if multiscale)?

### ISSUE THREAD

**Issue by RossTntt** [deneb2016/WSDDN.pytorch#2]
Why after train and the eval results in VOC 2007 just 14(18 if multiscale). I did not change the code

**Comment by YuanZhichao** [comment]
How did you run the train and  eval code? I guess you may not using the parameter `--use_prop_score`, which can boost the performance of the network.

**Comment by RossTntt** [comment]
> How did you run the train and eval code? I guess you may not use the parameter `--use_prop_score`, which can boost the performance of the network.

you are right, I do not use '--use_prop_score'. I read the paper of WSDDN, I don't think this will lead the mAP result as bad as 14. This result is too far below the mAP = 30+ in the paper.

**Comment by RossTntt** [comment]
> How did you run the train and eval code? I guess you may not using the parameter `--use_prop_score`, which can boost the performance of the network.

could you tell me your runing result??

**Comment by CCC-123** [comment]
HI, do you get the result of ma=30% now? I still can't get the good result and may need some help.

**Comment by njustymk** [comment]
me too , i only get the result map 16%

**Comment by HustHB** [comment]
You can change the scores' threshold to 0.01 or 0 when you want to calculate mAP.

**Comment by HustHB** [comment]
When I set the scores' threshold to 0.1, I also get a result of mAP=18% or so.

**Comment by HustHB** [comment]
However, I set the scores‘ threshold to 0, I only get mAP=23.91% that is far from 30%+
